### DISCUSSION TITLE
Commits count is not beeing shown on commit history link

### DISCUSSION THREAD

**Discussion by beqabeqa473** [community/community#80346] [Accessibility]
### Select Topic Area

Bug

### Body

Hello.

In the latest rewamp of repository pages, a regression was discovered, which doesn't allow users with screenreaders to get all commits count information of a particular repository.
The problem is that a link of commits history has an aria-label attribute, which overwrites a name for accessible object of assistive technologies, which should be a text of an inner span.
Please remove an attribute to make it possible to view commits count again.

Thanks.

**Comment by ghostinhershell** [comment]
Hey there @beqabeqa473, 

Thanks so much for writing in and thanks so much for reporting this issue. I will go ahead and report this to our Accessibility team for review. Can you share any additional information, like reproduction steps, so that I can share that with the team as well? Anything that would help clarify the issue. 

Thanks!

**Reply by beqabeqa473** [reply]
Hello. Try to open some repo page, and read a link, where commits count is visible for example using NVDA screenreader. It will read Commit history, because this particular element has aria-label set, which overwrites text from span, which is inner this element.

**Comment by beqabeqa473** [comment]
@ghostinhershell Thank you very much for so swift resolution of problem.
It is mutch easier to see commits count for now without extra manipulations.

Thanks.

**Comment by beqabeqa473** [comment]
@ghostinhershell Hello, displaying commits history is broken again. Issue is still active.

**Reply by cehfisher** [reply]
Hello @beqabeqa473 - thank you for providing this feedback! We will re-file this with our Accessibility team so that it can be reviewed and will follow-up here with any additional questions or updates. Thanks again!

**Reply by ericwbailey** [reply]
Hi there, @beqabeqa473. I have just merged a fix for this issue, so the commit count on the link immediately after the History heading on a repo page should now be restored.

You may need to refresh your browser tab to make these changes take effect. Thank you for reporting this, and your patience with a fix. Please let me know if the issue persists, and I'll make sure it gets addressed.
